projects/aca-folder-rules/src/lib/rule-set-picker/rule-set-picker.smart-component.html

```diff
@@ -0,0 +1,59 @@
+<div mat-dialog-title class="aca-rule-set-picker__header">
+  <div class="aca-rule-set-picker__header__title">
```

Comment:
Not sure if you want to cover accessibility problems here but I think that if you run axe tool then you will get issue that you should to use h1 tag because you need some header on page.